src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/ApiParameterDescriptionExtensions.cs

```diff
@@ -21,6 +21,13 @@ public static class ApiParameterDescriptionExtensions
 #endif
         };
 
+        private static readonly List<string> IllegalHeaderParameters = new List<string>
+        {
+            "accept",
+            "content-type",
+            "authorization"
+        };
```

Comment:
A `HashSet<T>` might be more performant here as all that's ever done is a `Contains()`. You can also then pass `StringComparer.OrdinalIgnoreCase` to the constructor and not need it for the `Contains()` call.

Also, if possible, consider using the [`HeaderNames`](https://learn.microsoft.com/dotnet/api/microsoft.net.http.headers.headernames) values, rather than hard-coding.